systemtest/src/main/java/io/strimzi/systemtest/utils/kubeUtils/controllers/DeploymentUtils.java

```diff
@@ -49,6 +47,30 @@ public static void replaceInNamespace(String namespaceName, String resourceName,
         KubeResourceManager.get().kubeClient().getClient().resources(Deployment.class, DeploymentList.class).inNamespace(namespaceName).resource(toBeReplaced).update();
     }
 
+    /**
+     * Returns {@link Deployment} from specified Namespace with specified name.
+     *
+     * @param namespaceName     Namespace name where the Deployment should be present.
+     * @param deploymentName    Name of the desired Deployment.
+     *
+     * @return  {@link Deployment} from specified Namespace with specified name.
+     */
+    public static Deployment getInNamespace(String namespaceName, String deploymentName) {
```

Comment:
I think simple `get()` would be enough. We agreed within TF that we won't allow a creation of method without namespace specification (only for cluster-wide resources) so every such method should be for a specific namespace specified as a parameter and I don't think we need it also in name.

Also it is a little bit strange to me to use `*Utils` classes for client operations. I think we talked about creating classes like `DeploymentCLients` and we agreed it doesn't make sense, but now....having getters implemented in utils looks strange to me. I wonder what others think about it. @kornys @see-quick

Reply:
As I mentioned offline, I would add clients encapsulation (in some way) into the `CrdClients` class (but I would change the name) and instead of implementing these kind of methods, use the clients directly from there. IMO the `*Utils` classes are really just garbage classes where everything we use once is implemented, so having these getters and other methods there doesn't seem much clear for me as well. But it was better than adding these methods to one yet another garbage class, as it was before in the `KubeClient` class.

Reply:
> I think simple get() would be enough. We agreed within TF that we won't allow a creation of method without namespace specification (only for cluster-wide resources) so every such method should be for a specific namespace specified as a parameter and I don't think we need it also in name.

Yeah, this sounds fair. (using just simple get())

> Also it is a little bit strange to me to use *Utils classes for client operations. I think we talked about creating classes like DeploymentCLients and we agreed it doesn't make sense, but now....having getters implemented in utils looks strange to me. What others think about it? @kornys @see-quick

Well, it's just a matter of tradeoff. I don't have a strong opinion on this, but I am fine having this inside `*Utils`.